src/app/shared/search-popup/search-popup.component.ts

```diff
@@ -1,6 +1,5 @@
 import { NewsSearchModel } from '@global-models/search/newsSearch.model';
 import { EventsSearchModel } from '@global-models/search/eventsSearch.model';
-import { SearchDataModel } from '@global-models/search/search.model';
```

Comment:
You can also delete unused fields `isNewsSearchFound, searctabindex`.(39, 43)
And add type `searchValueChanges: Observable<string>;`(40)